content/en/monitors/guide/scoping_downtimes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Scoping downtimes schedules
+kind: Guide
+disable_toc: false
+further_reading:
+- link: "/monitors/notify/downtimes"
+  tag: "Documentation"
+  text: "Downtimes Overview"
+- link: "/monitors/guide/suppress-alert-with-downtimes"
+  tag: "Guide"
+  text: "Suppress Alerts through the Downtimes API and UI"
+---
+
+## Overview
+
+Use group scope to apply additional filters to your downtime schedule and have more control over which monitors to mute. 
+
+The examples in this guide show how the `Group scope` may be applied to multi alert monitors.
+
+## By Monitor Name
+
+### Mute notification for a specific service
+
+1. To schedule downtime on only one group (in this case, `service:web-store`), enter that group in the `Group scope` field.
+2. **Preview affected monitors** indicates that the monitor chosen is still in scope, so alerts for the group `service:web-store` are muted during the scheduled downtime.
+
+{{< img src="monitors/downtimes/downtime_examplebyname1_downtime.jpg" alt="downtime example of 'By Monitor Name' with preview of affected monitors" style="width:80%;">}}
+
+3. Scheduled downtime begins, with only alerts for the group `service:web-store` muted for this monitor.
+
+{{< img src="monitors/downtimes/downtime_examplebyname1_monitor.jpg" alt="Evaluation graph showing downtime for group service:web-store" style="width:80%;">}}
```

Comment:
This is outside the scope of the ticket, but I feel like this section could be reworked. Steps 2 and 3 aren't really steps in the sense of completing a task.

Reply:
Thanks @drichards-87 I'll add this as a task in the [DOCS-5318](https://datadoghq.atlassian.net/browse/DOCS-5318)